src/components/slider/index.js

```diff
@@ -370,14 +438,43 @@ export default class Slider extends PureBaseComponent {
 
   /* Renders */
 
+  renderThumb = () => {
+    const {
+      thumbStyle,
+      disabled,
+      thumbTintColor
+    } = this.getThemeProps();
+    return (
+      <Animated.View
+        ref={this.setThumbRef}
+        onLayout={this.onThumbLayout}
+        {...this._panResponder.panHandlers}
```

Comment:
Moving the panHandlers here means you must touch and drag the thumb itself and not the track. Did you test this on a real device? Is it an easy thing to do? Is the hit slop big enough?

Reply:
Added.
You are right, although it's seems easy to drag it, i've added hitslop for better experience.

Reply:
One more thing about this. This is a UX change and I think we should pass it by our UX guys. Once they'll approve it we can merge.

Reply:
Approved by Yulia 🚀